_includes/components/sidebar.html

```diff
@@ -2,16 +2,21 @@
   <header class="site-header">
 
     <div class="lh-wrapper">
-      <img src="{{site.baseurl}}/assets/images/logo.jpg" alt="Accessibility: Advocate Equal Access">
       <a href="{{ '/' | relative_url }}" class="site-title lh-tight">
-        {{ site.title}}</a>
+        <img src="{{site.baseurl}}/assets/images/logo-wa11ypuu.png" alt="WP Accessibility logo, to homepage">
```

Comment:
I don't personally like having the accessible name of the link have all this information in it. This becomes "WP Accessibility logo, to home page WP Accessibility Knowledge Base", which seems excessive to me.